app/(candidate)/dashboard/contacts/[[...attr]]/components/ContactSearch.js

```diff
@@ -0,0 +1,24 @@
+'use client'
+
+import SearchInput from 'app/(candidate)/dashboard/issues/components/search/SearchInput'
+import { useState } from 'react'
+
+export const ContactSearch = () => {
+  const [searchText, setSearchText] = useState('')
+
+  // TODO in follow-up:
+  // - add a search icon
```

Comment:
We don't need this comment line anymore, right?

Reply:
fixed.